solc/CommandLineInterface.cpp

```diff
@@ -336,36 +338,50 @@ void CommandLineInterface::handleOpcode(string const& _contract)
 
 void CommandLineInterface::handleIR(string const& _contractName)
 {
-	if (m_args.count(g_argIR))
+	if (!m_args.count(g_argIR)
```

Comment:
I'm a bit puzzled by this - when I check out the branch locally I get the additional ``)`` that's missing here and apparently is also missing from CI runs... I could probably just rebase this and push, but I'd actually like to know what's happening here...

Reply:
Yea I also see `))` when I checkout the branch locally

Reply:
This is the archlinux build CI run at the time of writing https://circleci.com/gh/ethereum/solidity/266011?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link
So that CI run fails because of the missing ``)`` like displayed on github... I'll now retrigger that CI run to see, if it changes.

Reply:
This is the CI run after retriggering: https://circleci.com/gh/ethereum/solidity/266633?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

I SSH'd into it and checked: it has ``CIRCLE_SHA1`` (which is the PR commit hash) set to ``222dccd3338293af351eca541fda2082d8d51d18``, which does not seem to exist in the repo. Locally the head of the PR branch is at ``bc3261936471b0cc4f7b2c9ae1be3179d52ca71d`` for me.

And possibly github displays this non-existing commit in the diff...

Reply:
When attempting to create a new PR from the same branch, github shows the correct diff and points to this PR as already existing (even though the diff here is incorrect and this PR apparently has an invalid commit head). Not sure, if we should just ignore this or actually report it to github and ask for an explanation...